src/qml/NightstandPage.qml

```diff
@@ -0,0 +1,143 @@
+        anchors.margins: Dims.l(15)
+
+        Column {
+            id: onOffSettings
+            anchors.fill: parent
+            LabeledSwitch {
+                //% "Nightstand enable"
+                text: qsTrId("id-nightstand-enable")
+                width: parent.width
+                height: Dims.l(20)
+                checked: nightstandEnabled.value
+                onCheckedChanged: nightstandEnabled.value = checked
+            }
+
+            Column {
+                width: parent.width
+                opacity: nightstandEnabled.value ? 1.0 : 0.4
+		enabled: nightstandEnabled.value
+                Label {
+                    //% "Nightstand Brightness"
+                    text: qsTrId("id-nightstand-brightness")
+                    font.pixelSize: Dims.l(6)
+                    verticalAlignment: Text.AlignVCenter
+                    wrapMode: Text.Wrap
+                }
+                IntSelector {
+                    width: parent.width
+                    height: Dims.l(20)
+                    stepSize: 10
+                    value: nightstandBrightness.value
+                    onValueChanged: nightstandBrightness.value = value
+                }
+            }
+
+            Column {
+                width: parent.width
+                opacity: nightstandEnabled.value ? 1.0 : 0.4
+		enabled: nightstandEnabled.value
+                Label {
+                    //% "Nightstand delay"
+                    text: qsTrId("id-nightstand-delay")
+                    font.pixelSize: Dims.l(6)
+                    verticalAlignment: Text.AlignVCenter
+                    wrapMode: Text.Wrap
+                }
+                IntSelector {
+                    width: parent.width
+                    height: Dims.l(20)
+                    stepSize: 5
+                    max: 30
+                    unitMarker: "s"
+                    value: nightstandDelay.value
+                    onValueChanged: nightstandDelay.value = value
+                }
+            }
+
+            Row {
+                width: parent.width
+                height: Dims.l(20)
+                opacity: nightstandEnabled.value ? 1.0 : 0.4
+		enabled: nightstandEnabled.value
+                Label {
+                    //% "select nightstand watchface"
+                    text: qsTrId("id-nightstand-watchface")
+                    font.pixelSize: Dims.l(6)
+                    verticalAlignment: Text.AlignVCenter
+                    width: parent.width * 0.7143
+                    wrapMode: Text.Wrap
+                }
+                IconButton {
+                    id: nextButton
+                    iconName: "ios-arrow-dropright"
+                    height: parent.height
+                    width: height
+                    onClicked: layerStack.push(nightstandWatchfaceLayer)
+                }
+            }
+
+            Item { width: parent.width; height: Dims.l(10) }
```

Comment:
(out of scope for this PR don't worry, just a note) probably, this kind of spacing should be handled by qml-asteroid or we'd end up with inconsistencies, for example I suppose you introduced this for round watches but probably this doesn't make sense in square watches and it should be disabled there. But let's keep it that way

Reply:
I actually tested on both `sturgeon` and `dory` and thought it looked better this way even on the square `dory` screen.